db/migrate/20171222114708_add_sessions_table.rb

```diff
@@ -0,0 +1,12 @@
+class AddSessionsTable < ActiveRecord::Migration[5.1]
+  def change
+    create_table :sessions do |t|
```

Comment:
__[Reek]__

[UncommunicativeVariableName] AddSessionsTable#change has the variable name 't'


<!-- Comment by [SideCI](https://sideci.com) -->